infrastructure/src/main/java/mops/controllers/DozentEventController.java

```diff
@@ -88,6 +89,21 @@ public String handleFileUpload(@RequestParam("file") MultipartFile file,
     return "redirect:/feedback/dozenten/event/{veranstaltungsNr}";
   }
 
+  @PostMapping("/event/addStudent/{veranstaltungsNr}")
+  @RolesAllowed(ORGA_ROLE)
+  public String addStudent(@PathVariable Long veranstaltungsNr,
+                           RedirectAttributes redirectAttributes,
+                           String newStudent) {
+    Student student = new Student(newStudent);
+    Veranstaltung veranstaltung = veranstaltungen.getVeranstaltungById(veranstaltungsNr);
+    veranstaltung.addStudent(student);
+    veranstaltungen.save(veranstaltung);
```

Comment:
You don't need to save the event again. It's already saved. The changes are saved without this method call.